integration-tests/elytron-security-oauth2/src/main/java/io/quarkus/it/elytron/oauth2/MyResource.java

```diff
@@ -0,0 +1,33 @@
+package io.quarkus.it.elytron.oauth2;
+
+import javax.annotation.security.RolesAllowed;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+
+@Path("/api")
+@Produces(MediaType.APPLICATION_JSON)
+public class MyResource {
```

Comment:
@loicmathieu When you get few secs, can you please rename this resource (and the tests) to something more relevant as per the general recommendation by @gsmet, example, `ElytronOauth2ExtensionResource` or something better if you prefer

Reply:
done